### ISSUE TITLE
Read QR code and load a 3d asset

### ISSUE THREAD

**Issue by ardian-c** [viromedia/viro#822]
Hi, 

After going through some of the documentation, issues, and browsing I am still unsure if it is possible to use this library to show a 3d asset when the camera encounters a QR code in a map image. Any help, tips are appreciated. 

Thank you!

**Comment by HedwigAR** [comment]
Depends on if you know the QR code beforehand. I don't think there's a way to scan QR codes, but you could load it as a normal image and use that as a marker. https://docs.viromedia.com/docs/viroarimagemarker

**Comment by ardian-c** [comment]
Yes all the QR codes will be generated beforehand, I just need to recognize them with a value and load a 3D object on that surface.
